### ISSUE TITLE
Записи в чистой конфигурации

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#1318]
Originally reported on Google Code with ID 1318

```
По прежнему в пакете Комплексная автоматизация присутствуют записи:

1) Бабушкина А А
2) Иванов Иван Иванович
```

Reported by `gs1994` on 2009-04-12 13:10:28

**Comment by gsbelarus** [comment]
```
Появились из настройки "Справочник нормируемых расходов". Оттуда же появляется
подразделение "Офис", банк "Белинвестбанк ОАО отд-е инвестиционного кредитования",
счет "1123234545".
В настройке 3 документа (USR$ACC_ADVER), 7 позиций (USR$ACC_ADVERLINE)
```

Reported by `flakekun` on 2009-04-13 08:33:44

**Comment by gsbelarus** [comment]
```

  По теме...
```

Reported by `NikolayUkleyko` on 2013-05-18 09:01:01
- **Status changed**: `Duplicate`
- **Merged into**: #3100
